automation/aws/aws_rbd_from_tag/aws_rbd_from_tag.pt

```diff
@@ -0,0 +1,265 @@
+name "AWS Rule-Based Dimension From Account Tags"
+rs_pt_ver 20180301
+type "policy"
+short_description "Creates and/or updates individual Rule-Based Dimensions based on AWS Account tags. See the [README](https://github.com/flexera-public/policy_templates/tree/master/automation/aws/aws_rbd_from_tag) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.0",
```

Comment:
`v0.1`

Going to start using Semantic Versioning for new policies [Teams thread](https://teams.microsoft.com/l/message/19:833373548e104af2a20b0216eda1ba7b@thread.skype/1677611086855?tenantId=91034d23-0b63-4943-b138-367d4dfac252&groupId=fb250818-e040-4a26-b207-61c3cd99fd6e&parentMessageId=1677610206124&teamName=Team%20Flexera%20One&channelName=Policy%20Support%20and%20Questions&createdTime=1677611086855&allowXTenantAccess=false)